seeker_test.go

```diff
@@ -26,15 +26,16 @@ func TestSeeker(t *testing.T) {
 	f.WriteString("Line 1\n")
 	f.WriteString("Line 2\n")
 
-	s := NewSeeker(f)
+	s := New(f)
 
 	if err = s.SeekToStart(); err != nil {
 		t.Fatal(err)
 	}
 
 	var str string
-	if err = s.ReadLine(func(buf *bytes.Buffer) {
+	if err = s.ReadLine(func(buf *bytes.Buffer) error {
```

Comment:
We do in mrT, see this line for seeker:
https://github.com/itsmontoya/seeker/pull/1/files/474907a3be1798e5ac976de8ee02e8e61c1942f0#diff-fddc5bba3483c0030216dceff265ae46R267